index.html

```diff
@@ -65,5 +80,294 @@ <h2 class="SupportingText">I'am a Software Developer</h2>
+      </div>
+
+      <div class="textsupportingtext3">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+      </div>
+
+      <div class="tag5">
+        <img src="images/Tag (5).png" alt="Html">
+        <img src="images/Tag (6).png" alt="Bootstrap">
+        <img src="images/Tag (7).png" alt="Ruby">
+      </div>
+
+      <div class="Action3">
+        <a href="#"></a><img src="images/Action.png" alt="actions">
+      </div>
+
+      <div class="Testseeproject2">
+        <a href="#"></a><img src="images/See project 2.png" alt="seeproject">
+      </div> 
+
+      <div class="MaskGroup5"> 
+        <img src="images/Img Placeholder (4).png" alt="MaskGroup">
+      </div>
+
+      <div class="TextTitlePost4">
+        <p>Profesional Art Printing Data</p>
+      </div>
+
+      <div class="textsupportingtext4">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+      </div>
+
+      <div class="tag6">
+        <img src="images/Tag (5).png" alt="Html">
+        <img src="images/Tag (6).png" alt="Bootstrap">
+        <img src="images/Tag (7).png" alt="Ruby">
+      </div>
+
+      <div class="Action4">
+        <a href="#"></a><img src="images/Action.png" alt="actions">
+      </div>
+
+      <div class="Testseeproject3">
+        <a href="#"></a><img src="images/See project 2.png" alt="seeproject">
+      </div> 
+
+      <div class="MaskGroup6"> 
+        <img src="images/Img Placeholder (6).png" alt="MaskGroup">
+      </div>
+
+      <div class="TextTitlepost5">
+        <p>Profesional Art Printing Data</p>
+      </div>
+
+      <div class="textsupportingtext5">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+      </div>
+
+      <div class="tag7">
+        <img src="images/Tag (5).png" alt="Html">
+        <img src="images/Tag (6).png" alt="Bootstrap">
+        <img src="images/Tag (7).png" alt="Ruby">
+      </div>
+
+      <div class="Action5">
+        <a href="#"></a><img src="images/Action.png" alt="actions">
+      </div>
+
+      <div class="Testseeproject4">
+        <a href="#"></a><img src="images/See project 2.png" alt="seeproject">
+      </div> 
+
+    </section>
+
+    <section class="About-me-section">
+      <div class="Primarytext">
+        <p>About me</p>
+      </div>
+
+      <div class="TextDescription">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      </div>
+
+      <div class="Enable">
+        <img src="images/Enable.png" alt="enable">
+      </div>
+
+      <div class="Grouplogo">
+        <img src="images/Group (1).png" alt="Grouplogo">
```

Comment:
- [ ] This image serves as the background image of the section. Since it only serves as a decoration, it should be added as a backgrounded image with CSS. and not `img` tags). You can read more about the Css Background Image property [here](https://www.w3schools.com/cssref/pr_background-image.php).